scot/ooapi.py

```diff
@@ -100,6 +106,12 @@ def __init__(self, var, locations=None, reducedim=None, nfft=512, fs=2, backend=
         except TypeError:
             self.var_ = var
 
+    def __repr__(self):
+        s = ("Workspace(var={}, locations={}, reducedim={}, nfft={}, fs={}, "
+             "backend={})".format(self.var_, self.locations_, self.reducedim_,
+                                  self.nfft_, self.fs_, self.backend_))
```

Comment:
I don't like the fact that a `dict` is returned if we specify a backend with a string, whereas otherwise a `BackendManager` is required. Do you have a suggestion how to streamline this?